packages/streamid/src/event-id.ts

```diff
@@ -0,0 +1,202 @@
+import { CID } from 'multiformats/cid'
+import { base36 } from 'multiformats/bases/base36'
+import { hash as sha256 } from '@stablelib/sha256';
+import varint from 'varint'
+import * as cbor from 'cborg'
+import * as u8a from 'uint8arrays'
+import { STREAMID_CODEC } from './constants.js'
+import { Memoize } from 'mapmoize'
+
+const TAG = Symbol.for('@ceramicnetwork/streamid/EventID')
+
+const DAG_CBOR_CODEC = 0x30
```

Comment:
```suggestion
const EVENT_IT_STREAM_TYPE = 0x30
```
This is no longer the dag cbor code.

Reply:
nit: I usually link to the table in a comment next to magic number so later readers know where the magic is coming from.
https://github.com/ceramicnetwork/CIPs/blob/main/tables/streamtypes.csv

Reply:
oh my PR is not merged I pointed you at a lying link. the number is going to be `0x05`
https://github.com/ceramicnetwork/CIPs/pull/139/files#diff-14311009753dce3ff50ded6e3fec3401bbd7e85f24071a0dba88bb390046bd13

Reply:
update value